### DISCUSSION TITLE
How to bring the route name above the LineLayer?

### DISCUSSION THREAD

**Discussion by mnismt** [rnmapbox/maps#3077] [Q&A] [answered]
Hi, thanks for the great work!

I’m using the LineLayer to draw some lines with the Directions API from Mapbox. The problem is that the route name of the map style is always below the LineLayer (you can see the image below):

![CleanShot 2023-09-26 at 09 52 46@2x](https://github.com/rnmapbox/maps/assets/27861064/e9f18dcf-5be3-4af7-ab83-1743b9b178db)

I want to display the line above the road but below the road name of the current map style. Is that possible?

Thanks for helping!

**Comment by mfazekas** [comment, marked as answer]
@mnismt I think in studio you should be able to get the layerid of the road names and then you can use belowLayerID on your LineLayer

**Reply by mnismt** [reply]
It works! Thanks a lot. The route name layer usually is `﻿road-label-navigation` in most styles for anyone who is looking for it.

```jsx
<Mapbox.LineLayer
  id={`${id}-base-line`}
  belowLayerID="road-label-navigation"
  style={{
    ...lineStyles,
    ...style,
  }}
/>
</Mapbox.ShapeSource>
```

![Screenshot_20230927-112938~2](https://github.com/rnmapbox/maps/assets/27861064/6ba8fb09-aad5-4ed8-b58f-fc2f44293df6)
